cmd/rook/block/mount.go

```diff
@@ -131,11 +131,6 @@ func mountBlock(name, poolName, mountPoint, rbdSysBusPath string, c client.RookR
 		return "", err
 	}
 
-	// format the device with a default file system
-	if err := sys.FormatDevice(devicePath, executor); err != nil {
```

Comment:
if no formatting of the new RBD device occurs, what happens in the next step of mounting?  If there's no filesystem on the device, what does mounting do? it succeeds?